openapi/horizon/bundled.yml

```diff
@@ -1163,6 +1163,145 @@ paths:
               examples:
                 SubmitaTransaction:
                   $ref: '#/components/examples/SubmitaTransaction'
+  /transactions_async:
+    post:
+      summary: Submit Async Transaction
+      description: This endpoint submits a transaction in an async manner to the Stellar network. It is designed to allow users to submit transaction without blocking them while waiting for a response from Horizon. At the same time, it also provides clear response status codes from stellar-core to help understand the status of submitted transaction. You can then use Horizon's [GET transaction endpoint](./retrieve-a-transaction.api.mdx) to check whether the transaction was successfully ingested or not.
+      operationId: SubmitAsyncTransaction
+      tags:
+        - Transactions
+      parameters:
+        - in: query
+          name: tx
+          schema:
+            type: string
+            description: The base64-encoded XDR of the transaction.
+          required: true
+      responses:
+        '201':
+          description: Transaction has been received by core and is in pending status.
```

Comment:
Maybe we should again point people to `GET /transactions/:id` to monitor its success or failure.

Reply:
@JakeUrban I dont think we should add that info here. I would like to have this description just describe the response status. We already mention checking the GET endpoint in the endpoint summary. What do you think?